lib/src/main/java/uk/gov/di/ipv/core/library/statemachine/BaseJourneyLambda.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.di.ipv.core.library.statemachine;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpStatus;
+import uk.gov.di.ipv.core.library.domain.JourneyResponse;
+import uk.gov.di.ipv.core.library.exceptions.HttpResponseExceptionWithErrorBody;
+import uk.gov.di.ipv.core.library.helpers.ApiGatewayResponseGenerator;
+import uk.gov.di.ipv.core.library.helpers.RequestHelper;
+
+import java.util.Map;
+
+public abstract class BaseJourneyLambda
```

Comment:
I'm curious as to the thinking here. Do you think we should have a base class after the refactor or is it only where we are moving things about.

Reply:
This is just for the refactor. Then I want it removed.